components/scream/tests/scream_zm/input.yaml

```diff
@@ -0,0 +1,17 @@
+%YAML 1.1
+---
+Debug:
+  Atmosphere DAG Verbosity Level: 5
+
+Atmosphere Processes:
+  Number of Entries: 1
+
+  Process 0:
+    Process Name: ZM
+    Grid: Physics
+    Can Initialize All Inputs: true
```

Comment:
You are not currently using this. But see my comment above on checking P3 approach on input initialization. Might be that you want to keep this option, and start using it, like P3 does.